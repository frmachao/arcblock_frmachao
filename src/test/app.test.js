import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from '../app';

afterEach(cleanup);

it('should take a snapshot', () => {
  // 渲染组件
  const { asFragment } = render(<App />);
  // 将渲染的组件和快照匹配
  expect(asFragment()).toMatchSnapshot();
});
