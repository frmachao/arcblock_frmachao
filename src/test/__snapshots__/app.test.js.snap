// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <header
      class="app-header"
    >
      <img
        alt="logo"
        class="app-logo"
        src="logo.svg"
      />
      <a
        class="app-link"
        data-testid="demo-link"
        href="/demo"
      >
        Demo Prower by machao
      </a>
      <a
        class="app-link"
        href="/about"
      >
        About
      </a>
      <a
        class="app-link"
        href="https://docs.arcblock.io/abtnode/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn Blocklet
      </a>
    </header>
  </div>
</DocumentFragment>
`;
